cmd/elastic.sh

```diff
@@ -62,7 +62,14 @@ function elastic_info(){ curl -s "http://${ELASTIC_HOST:-localhost:9200}/"; }
 register 'elastic' 'info' 'display elasticsearch version and build info' elastic_info
 
 function elastic_stats(){
-  curl -s "http://${ELASTIC_HOST:-localhost:9200}/pelias/_search?request_cache=true&timeout=10s&pretty=true" \
+  #Extract the API section from the pelias.json
+  api_section=$(cat "$DATA_DIR/../pelias.json" | sed -n '/"api": {/,/}/p')
+  #Extract the value of api.indexName
+  index_name=$(echo "$api_section" | grep '"indexName"' | sed 's/.*"indexName": "\(.*\)".*/\1/')
```

Comment:
I'd prefer to avoid using `sed` if possible, `jq` would be good option if it didn't introduce an external dependency. maybe `node -e` is the best option for this and the above?

Reply:
This works, but only when the `node_modules` directory is present, which it isn't for the docker bash scripts 😿 

```bash
node -e "console.log(require('pelias-config').generate().schema.indexName)"
```

Reply:
Maybe it's best to move this into the `pelias/schema` module and then call it through docker?